corehq/apps/settings/views.py

```diff
@@ -171,7 +172,7 @@ def process_add_phone_number(self):
             user = self.request.couch_user
             user.add_phone_number(self.phone_number)
             user.save()
-            messages.success(self.request, _("Phone number added."))
+            messages.success(self.request, _("Phone number added!"))
```

Comment:
I'm not a huge fan of this, as it kind of makes it seem like we are surprised that we managed to add a phone number. This will also mean that if these strings are already translated, they will need to be translated again.

Reply:
Found multiple places with the exclamation points, I changed them to periods, is it worth it to do this in new places if it will need to be translated again?
https://github.com/dimagi/commcare-hq/pull/19574/commits/8ea443320564bbe5141c9cc89de597413df3f1ab